express/__init__.py

```diff
@@ -90,7 +92,6 @@ def property(self, property_name: str, *args, **kwargs) -> dict:
         Returns:
              dict
         """
-        Property_Class = self._get_property_class(property_name)
-        # This is an instance or child of BaseProperty
-        property_instance = Property_Class(property_name, self.parser, *args, **kwargs)
+        Property_Class: Type[BaseProperty] = self._get_property_class(property_name)
+        property_instance: BaseProperty = Property_Class(property_name, self.parser, *args, **kwargs)
```

Comment:
you probably don't have to be this zealous about the types, it's pretty clear that calling a thing with type `Type[T]` will return a thing with type `T`.  It's a balance between being explicit with types, and being overly wordy.

Reply:
Redundant typehints have been removed